Makefile

```diff
@@ -98,7 +98,7 @@ fmt: ## Run go fmt against code.
 	go fmt ./...
 
 .PHONY: vet
-vet: ## Run go vet against code.
+vet: ## Run go vet against code.sadsada
```

Comment:
:racehorse:

Reply:
how did this appear here?

Reply:
oh i looked a the lines and thought it was removed. :rofl: